.travis.yml

```diff
@@ -5,38 +5,22 @@ addons:
     packages:
       - w3m
 
-env:
-  global:
-    - ANDROID_TARGET=android-22
-    - ANDROID_ABI=armeabi-v7a
-    - ADB_INSTALL_TIMEOUT=12 # in minutes
-
 jdk:
-  # - openjdk8 # not yet available
   - oraclejdk8
 
 android:
   components:
     - tools
     - platform-tools
-    - build-tools-27.0.0
+    - build-tools-27.0.3
     - extra-google-m2repository
     - extra-android-m2repository
-    - ${ANDROID_TARGET}
-    - android-25
-    - android-26
     - android-27
-    - sys-img-${ANDROID_ABI}-${ANDROID_TARGET}
   licenses:
     - 'android-sdk-license-.+'
 
-before_script:
-  - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi $ANDROID_ABI
-  - emulator -avd test -no-audio -no-window -no-boot-anim &
-  - android-wait-for-emulator
-
 script:
-  - ./gradlew clean check connectedCheck jacocoTestReport
```

Comment:
I believe this change removes connected tests and code coverage checks (tests under app/src/androitTests) from Travis build scripts. Why do they have to go?

Reply:
@whym  whoops, I've put this back in the subsequent commits. I think I must have removed this while getting stuff working locally.